### PR TITLE
Handle MultiLineString with empty coordinates

### LINKED ISSUE #163: Unable to decode WKB of MultiLineString with no coordinates
Geo is currently unable to decode a MultiLineString that has an empty list of coordinates (although it is able to encode it):
```
iex(8)> Jason.decode!(~s/{"type":"MultiLineString","coordinates":[[]]}/)
|> Geo.JSON.decode!()
|> Geo.WKB.encode!()
|> Geo.WKB.decode!()
** (FunctionClauseError) no function clause matching in anonymous fn/2 in Geo.WKB.Decoder.do_decode/4

    The following arguments were given to anonymous fn/2 in Geo.WKB.Decoder.do_decode/4:

        # 1
        1

        # 2
        ""

    (geo 3.4.2) lib/geo/wkb/decoder.ex:108: anonymous fn/2 in Geo.WKB.Decoder.do_decode/4
    (elixir 1.12.3) lib/enum.ex:1710: anonymous fn/3 in Enum.map_reduce/3
    (elixir 1.12.3) lib/enum.ex:3950: Enum.reduce_range/5
    (elixir 1.12.3) lib/enum.ex:2389: Enum.map_reduce/3
    (geo 3.4.2) lib/geo/wkb/decoder.ex:108: Geo.WKB.Decoder.do_decode/4
    (geo 3.4.2) lib/geo/wkb/decoder.ex:235: anonymous fn/2 in Geo.WKB.Decoder.do_decode/4
    (elixir 1.12.3) lib/enum.ex:1704: Enum."-map_reduce/3-lists^mapfoldl/2-0-"/3
    (geo 3.4.2) lib/geo/wkb/decoder.ex:234: Geo.WKB.Decoder.do_decode/4
    (geo 3.4.2) lib/geo/wkb.ex:82: Geo.WKB.decode!/1
```

The WKB that is generated is `"000000000500000001000000000200000000"`, which appears to be valid and on an online conversion website (https://rodic.fr/blog/online-conversion-between-geometric-formats/) it decodes to `{"type":"MultiLineString","coordinates":[[]]}`.

### PR DESCRIPTION
**Pull request by axelson**
Fixes #163

### CODE REVIEW

File: test/geo/wkb_test.exs [hunk 1]
```diff
@@ -399,6 +399,13 @@ defmodule Geo.WKB.Test do
     assert(point.srid == 4326)
   end
 
+  test "Decode EWKB of empty MultiLineString" do
+    multi_line_string = Geo.WKB.decode!("010500000001000000010200000000000000")
+    assert multi_line_string.coordinates == [[]]
+    assert multi_line_string.properties == %{}
+    assert multi_line_string.srid == nil
```
Comment (cduruk) [inline test/geo/wkb_test.exs:406]: should we add a test to make sure the SRID is passed through too?

Comment (axelson) [reply test/geo/wkb_test.exs:406]: Good idea. Just added it in dfe665fc0a45a2d7343a74e08046322464600b45

**Comment by bryanjos** [comment]
Sorry I'm just getting to this. And thanks!

**Comment by axelson** [comment]
Thank you for merging this! And also for maintaining the library!

**Pull request merged**